packages/cli-hermes/src/profileHermes/index.ts

```diff
@@ -0,0 +1,79 @@
+import {logger} from '@react-native-community/cli-tools';
+import {Config} from '@react-native-community/cli-types';
+import {downloadProfile} from './downloadProfile';
+
+type Options = {
+  verbose: boolean;
```

Comment:
No need to define `verbose` in `Options` type, as we already do that in the main file:
https://github.com/react-native-community/cli/blob/b7778db91e1033e8f5a6338f2e8805c4b6f6d967/packages/cli/src/index.ts#L20